kbs/docker/Dockerfile.rhel-ubi

```diff
@@ -22,7 +22,12 @@ cargo install --locked --root /usr/local/ --path kbs/src/kbs --no-default-featur
 mkdir -p /root/trustee/lib64 && \
 ldd /usr/local/bin/kbs | sed 's@.*\s/@/@' | sed 's/\s.*//' | xargs -I {} cp {} /root/trustee/lib64
 
-# Package minimal image.
-FROM registry.access.redhat.com/ubi9-micro
+# Package UBI image.
+FROM registry.access.redhat.com/ubi9
+
+# Install runtime dependencies from Intel repo.
+COPY --from=builder /root/sgx_rpm_local_repo /root/sgx_rpm_local_repo
```

Comment:
perhaps remove the local repo after the necessary packages are installed to reduce the final image size a bit?